lesson_14.com.ua/homework_14_csv.py

```diff
@@ -0,0 +1,43 @@
+import csv
+import os
+
+# Определяем пути к файлам
+base_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+csv_folder = os.path.join(base_path, 'ideas_for_test', 'work_with_csv')
+file1_path = os.path.join(csv_folder, 'r-m-c.csv')
+file2_path = os.path.join(csv_folder, 'rmc.csv')
+output_file_path = os.path.join(os.path.dirname(__file__), 'result_kotsar.csv')
+
+
+# Функция чтения CSV файла и возврата списка строка
+def read_csv(file_path):
+    with open(file_path, newline='', encoding='utf-8') as csvfile:
+        reader = csv.reader(csvfile)
+        return list(reader)
```

Comment:
```suggestion
        return reader
```
там і так повертає ліст за замовчуванням